protocol/swap/types_test.go

```diff
@@ -27,8 +27,12 @@ func Test_InfoMarshal(t *testing.T) {
 		200,
 		make(chan types.Status),
 	)
+	err := info.StartTime.UnmarshalJSON([]byte("\"2023-02-20T17:29:43.471020297-05:00\""))
```

Comment:
You'd need 2 lines, but this is probably more elegant:
```
	var err error
	info.StartTime, err = time.Parse(time.RFC3339, "2023-02-20T17:29:43.471020297-05:00")
```
See here:
https://cs.opensource.google/go/go/+/refs/tags/go1.19.6:src/time/time.go;l=1360